ubyssey/static/src/styles/modules/events/_event.scss

```diff
@@ -64,6 +64,7 @@
 
 // EventBox
 .c-event--box {
+  display: inline-block;

```

Comment:
This should actually be `display: block;`. The default for `a` tags is `inline-block`, but we want this to behave like a `div`, which has `block` positioning. 

```scss
.c-event--box {
  // Structure
  display: block;